test/cmd/status.sh

```diff
@@ -23,6 +23,7 @@ os::test::junit::declare_suite_start "cmd/status"
 # login and ensure no current projects exist
 os::cmd::expect_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority='${MASTER_CONFIG_DIR}/ca.crt' -u test-user -p anything"
 os::cmd::expect_success 'oc delete project --all'
+os::cmd::try_until_text "oc get projects -o jsonpath='{.items}' 2>/dev/null" "^\[\]$"
```

Comment:
Happens with any kind of output format and resource when there is nothing returned from the server. Does not seem broken to me since it is printed on stderr, not impacting scriptability / consumption of machine parseable output